client/ui/controllers/systemController.cpp

```diff
@@ -24,6 +24,20 @@ SystemController::SystemController(const std::shared_ptr<Settings> &settings, QO
 {
 }
 
+void SystemController::setHasFocus(bool hasFocus)
```

Comment:
I think these functions should be in the PageController

Reply:
Renamed hasFocus (and all functions and all signals) to appHasFocus.